### PR TITLE
Allow scout to double jump when bhoping

### PR DESCRIPTION
**Pull request by nepcat**

### CODE REVIEW

**Comment by 8dcc** [comment]
Just as a side note, on Nov 12, 2023 I tried adding this but was very buggy:


https://github.com/8dcc/tf2-cheat/assets/29655971/646ea12f-a6e9-46fa-bebf-bb63176d8d46

Now I just realized that it was probably fixed in commit d9fcfb5ff2ce9fba9e6aee73fc56a4bfd6a79ac3

**Comment by nepcat** [comment]
Is there anything wrong with my pull request?

**Comment by 8dcc** [comment]
> Is there anything wrong with my pull request?

Yes, I made a review explaining that line 75 should remain the same. I can make the changes if you want, I was waiting for you.

**EDIT**: I just realized I can only see the review if I am logged in, not sure why.

**Review by 8dcc — CHANGES_REQUESTED** [review]

File: src/features/movement.c [hunk 1]
```diff
@@ -64,14 +64,15 @@ void bhop(usercmd_t* cmd) {
...
+    const bool is_on_ground     = (g.localplayer->flags & FL_ONGROUND) != 0;
 
-    /* TODO: Because we disable jumping mid-air, this blocks scout's double
-     * jump. I tried SEOwned's method, but it only works ~1/5 of the time */
-    if (!(g.localplayer->flags & FL_ONGROUND))
+    if (last_is_jumping && !is_on_ground)
         cmd->buttons &= ~IN_JUMP;
 
-    if (is_jumping) {
+    if (!is_on_ground) {
```
[the remark this answers is not in the document]
Comment (nepcat) [reply src/features/movement.c]: Done

**Review by 8dcc — APPROVED** [review]

**Comment by 8dcc** [comment]
Thank you for your contribution!

**Pull request merged**